flatpak-version.yml

```diff
@@ -0,0 +1,7 @@
+# no-commit
+# This file is generated by utils/generate-version as part of the
+# Makefile / CI system. Please do not commit it.
```

Comment:
Looks like you did commit it.

Reply:
:man_facepalming: curse you not running my pre-commit hooks, gitg!